.github/workflows/windows.yml

```diff
@@ -1,9 +1,5 @@
 name: Windows Tests
```

Comment:
It looks like master will still run this workflow (and master will soon have the HostProcess code which no longer creates a binary), what's the plan to test HostProcess in master and client/server in v1.x?

Reply:
I think the idea is to (a) merge this change in master and then merge it into v1.x (b) slowly add the v2 changes in while keeping v1 around (so the client binary is still there) and (c) delete v1 and the associated v1 GitHub action so that only v2 tests run. During (b) this current test should also run added v2 tests and still pass

Reply:
sounds like a good plan